echopype/tests/echodata/test_zarr_combine.py

```diff
@@ -237,7 +238,7 @@ def test_append_ds_list_to_zarr(append_ds_list_params):
 
     # create temporary directory for zarr store
     temp_zarr_dir = tempfile.TemporaryDirectory()
-    zarr_path = temp_zarr_dir.name + "/" + zarr_name
+    zarr_path = Path(temp_zarr_dir.name + "/" + zarr_name)
```

Comment:
Please add a test that does same thing on minio.

Reply:
@lsetiawan I believe that my most recent changes make it unnecessary to add a test on minio. Please confirm.

Reply:
We'll just revisit this addition when working in making more tests. I don't remember if there's a zarr_combine test that hit minio.